libiocage/lib/helpers.py

```diff
@@ -102,6 +102,26 @@ def validate_name(name):
     return bool(validate.fullmatch(name))
 
 
+# data -> bool | default
+# This little helper function will try to parse booleans from strings
+# On success, it returns the parsed boolean, on failure it returns the
+# `default`. By default, `default` is `False`.
+def parse_bool(data, default=False):
```

Comment:
In addition it's odd to say it returns the default which is False. Why not just say else it returns False?